LOAN/RealEstateLoans/ConstructionLoans.rdf

```diff
@@ -41,140 +37,89 @@
-		<rdfs:label>on frame modular</rdfs:label>
-		<skos:definition>a factory built dwelling on a permanent chassis which does not have a HUD label</skos:definition>
-	</owl:NamedIndividual>
-	
-	<owl:NamedIndividual rdf:about="&fibo-loan-reln-cnst;ConstructionType_SiteBuilt">
-		<rdf:type rdf:resource="&fibo-loan-reln-cnst;ConstructionType"/>
-		<rdfs:label>site built</rdfs:label>
-		<skos:definition>describes construction process, indicating that most elements are created at the homes permanent site. May include some prefabricated components</skos:definition>
-	</owl:NamedIndividual>
-	
 	<owl:ObjectProperty rdf:about="&fibo-loan-reln-cnst;hasMaximumAllowedBalance">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-dae-dbt;hasAvailableAmount"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-pas-caa;hasBalance"/>
 		<rdfs:label xml:lang="en">has maximum allowed balance</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-reln-cnst;ConstructionLoanContract"/>
+		<rdfs:domain>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-loan-reln-cnst;ConstructionLoan">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-dae-dbt;SubFacility">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:domain>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-		<skos:definition xml:lang="en">Maximum balance - For loans with flexible re-draw facilities, the maximum balance that could potentially be outstanding based on a credit limit</skos:definition>
+		<skos:definition xml:lang="en">for loans with flexible re-draw facilities, the maximum balance that may potentially be outstanding at any time below the credit limit</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-loan-reln-cnst;hasMaximumAnticipatedBalance">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-dae-dbt;hasAvailableAmount"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fbc-pas-caa;hasBalance"/>
 		<rdfs:label xml:lang="en">has maximum anticipated balance</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-reln-cnst;ConstructionLoanCreditFacilityTranche"/>
+		<rdfs:domain>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-loan-reln-cnst;ConstructionLoan">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-dae-dbt;SubFacility">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:domain>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
-		<skos:definition xml:lang="en">Maximum balance - For loans with flexible re-draw facilities, the maximum balance the borrower believes they might need, below the credit limit, for planning purposes</skos:definition>
+		<skos:definition xml:lang="en">for loans with flexible re-draw facilities, the maximum balance the borrower believes they might need, below the credit limit, for planning purposes</skos:definition>
```

Comment:
why is "maximum" needed when there's a separate property with the (real) maximum? Does it also need to be below (or equal to?)  the maximum allowed balance?

Reply:
There are sub-facilities, each of which can have balances, maximum balances, etc., and then there is the total maximum for the facility at the highest level. Again these came from some of the reporting / analysis documents, but we can refine this ontology further in another iteration. It's provisional. This was there before but I've fixed some of the issues with it based on legislation / reporting review. All three properties were there from what either Mike or Michael had done originally.